tests/transactional/transactional_update.pm

```diff
@@ -75,10 +78,10 @@ sub run {
 
     get_utt_packages;
 
-    record_info 'Install ptf', 'Install package - snapshot #1';
-    trup_call "ptf install" . rpmver('security');
+    record_info 'Install pkg', 'Install package - snapshot #1';
+    trup_call "pkg install" . rpmver('security');
```

Comment:
Hmm, not sure if that's ok as ptf is changes what we test here. Any reason to change it?

Reply:
Seems to be the same thing and probably more updated is to use `pkg` because I didn't find it in the --help of the command. It works with both as far as I could test. Here I found some doc https://github.com/openSUSE/transactional-update/blob/b2e191275b70d4c4c0e717498a914dcb156121db/sbin/transactional-update.in#L568

Reply:
PTF is a specific kind of package. Please, don't change the meaning of the tests.
It is bad documented but a PTF is a package that follows specific requirements, it is a hotpatch sent to customers before the bug is addressed in a proper update channel. The test has to cover PTF. So, even it they seems to behave similar, the goal of the test is not testing all kind of packages, but PTF's.

Reply:
@SergioAtSUSE  thumb down means that you think is not? should I change it back?

Reply:
addressed.